README.md

```diff
@@ -42,6 +43,7 @@ Setup environment for building the app
 
 Fetch the sources and build the Zilliqa Ledger Nano-S app. You should find `app.hex` in the `bin/` directory.
   - (ledgerenv) ...`$cd $LEDGER_DIR; git clone https://github.com/Zilliqa/ledger-app-zilliqa`
+  - (ledgerenv) ...`export SCRIPT_LD=${LEDGER_DIR}/ledger-app-zilliqa/script.ld`
```

Comment:
I guess you can remove this which is now in the `Makefile` (see 583d129c365e3f81789f7e0b3755706c1a5f785c).

Reply:
> I guess you can remove this which is now in the `Makefile` (see [583d129](https://github.com/LedgerHQ/app-zilliqa/commit/583d129c365e3f81789f7e0b3755706c1a5f785c)).

Done